modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -799,8 +810,9 @@ public ObjectRelationship updateObjectRelationship(
 		}
 
 		if (objectRelationship.isReverse()) {
-			throw new ObjectRelationshipReverseException(
-				"Reverse object relationships cannot be updated");
+			return objectRelationshipLocalService.
+				updateReverseObjectRelationship(
+					externalReferenceCode, objectRelationship);
 		}
 
 		_validateExternalReferenceCode(
```

Comment:
This new method `updateReverseObjectRelationship` is only used here, so we could remove it and change the code to:

```
_validateExternalReferenceCode(
	externalReferenceCode, objectRelationshipId,
	objectRelationship.getCompanyId(),
	objectRelationship.getObjectDefinitionId1());

if (objectRelationship.isReverse()) {
	objectRelationship.setExternalReferenceCode(externalReferenceCode);

	return objectRelationshipPersistence.update(objectRelationship);
}

_validateParameterObjectFieldId(
```